### DISCUSSION TITLE
CORS Policy Problem SKyWalking APM Setting

### DISCUSSION THREAD

**Discussion by SteveLiu1218** [apache/skywalking#10578] [Q&A]
I get the error from my angular project

Access to XMLHttpRequest at 'http://127.0.0.1:8080//browser/perfData' from origin 'http://localhost:4200' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.

So I try to add the header to application.yaml (config/application.yaml)

server:
  cors:
    allowed-origins: "*"
    allowed-methods: GET, POST, PUT, DELETE
    allowed-headers: "*"
    allow-credentials: true

But Still get the error message again .
Does anyone can help ? I will appreciate that .

**Comment by wu-sheng** [comment]
cors is generally as same as other cases. Use a peoxy or add skywalking addr to the trust list.
